truss/templates/server/common/retry.py

```diff
@@ -0,0 +1,20 @@
+from typing import Callable
```

Comment:
why write this from scratch as opposed to using tenacity?
https://tenacity.readthedocs.io/en/latest/

Reply:
Do we need a backoff?

Reply:
We don't need backoff. Backoff is usually to reduce load on a downstream system. Not likely to be the case here.